federator/src/contracts/Constants.ts

```diff
@@ -0,0 +1,5 @@
+export const V4 = 'v4';
+export const V3 = 'v3';
+export const V2 = 'v2';
+export const V1 = 'v1';
+export const V0 = 'v0';
```

Comment:
it easier to export an object like
export const VERSIONS = {
"V4": "v4",
"V3": "v3",
"V2": "v2",
"V1": "v1",
"V0", "v0"
}